rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -693,6 +706,157 @@ void getOldFederationBtcUTXOs_calledTwice_returnCachedUtxos() {
 
     }
 
+    @Test
+    void getNewFederation_initialVersion() {
+        List<Integer> storageCalls = new ArrayList<>();
+        List<Integer> deserializeCalls = new ArrayList<>();
+        Federation newFederation = buildMockFederation(100, 200, 300);
```

Comment:
We have builders for this. P2shErpFederationBuilder and StandardMultiSigFederationBuilder.

I'd try to always use P2shErpFederationBuilder by default, as that is the current federation type. Unless testing some legacy conditions that requires the standard one

Reply:
Agree, replace it with the util fed builder.